tracks/ruby/exercises/series/mentoring.md

```diff
@@ -33,12 +34,13 @@ and to the rubydocs [Enumerable](https://ruby-doc.org/core/Enumerable.html) with
 
 ### Talking points
 - `each_cons` instead of an iterator `with_index`: In Ruby, you rarely have to write iterators that need to keep track of the index. Enumerable has powerful methods that do that for us.
-- `chars`: instead of `split('')`   
+- `chars`: instead of `split('')`. Or even better, `each_char`.   
```

Comment:
This could be another bullet point:

```suggestion
- `chars`: instead of `split('')`.
- `each_char`: if an `Array` is not specifically necessary or wanted.
```